### DISCUSSION TITLE
How to Connect Solana network with Wagmi?

### DISCUSSION THREAD

**Discussion by Nilbristi** [wevm/wagmi#3536] [Q&A]
Hello Developers,

I have used **@wagmi/core**  in my project. Now the client is asking for Solana network. I just want to know **how can I add Solana network with Wagmi**? If not then what is the alternative?

**Comment by tmm** [comment]
It's not possible to use Wagmi with Solana at this time.

**Comment by Nilbristi** [comment]
Hi, Wagmi Team

Does Wagmi have any plans to add SOLANA? If not, kindly try adding this feature as soon as you can; if not, we will have to switch to another product.
